### PR TITLE
formTitle added

### PR DESCRIPTION
**Pull request by eilusion**

### CODE REVIEW

File: src/components/FormTitle/FormTitle.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@import "./../../globalStyles/variables.scss";
+
+.formTitle {
+    display: flex;
+    background-color: $neutralPaper;
```
Comment (Cikica) [inline src/components/FormTitle/FormTitle.scss:5]: You dont need to set a background color, its justr a title comp it does not concern itself with what the color behind it is, thats the parents job.

File: src/components/FormTitle/FormTitle.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./FormTitle.scss";
+
+export const FormTitle = (props) => {
+    return (
+        <div className="formTitle">
+            <div className="formTitle__text">{props.text}</div>
```
Comment (Cikica) [inline src/components/FormTitle/FormTitle.jsx:7]: You dont need nesting here, its redundant, having `formTitle__text` nested inside `formTitle` does not help you achieve anything you could not have done by putting all the styles just on `formTitle` and deleting the `formTitle__text`

File: src/components/FormTitle/FormTitle.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+    &__text {
+        font-family: $barlow;
+        font-size: 24px;
```
Comment (Cikica) [inline src/components/FormTitle/FormTitle.scss:9]: Do we not have font size variables in the design system?
If we do you're goona need to carry em over.

File: src/components/FormTitle/FormTitle.jsx (hunk 2 above)
Comment (Cikica) [inline src/components/FormTitle/FormTitle.jsx:7]: We never talked about this, but its more valuable to allow a title component to render `props.chldren` rather than a specific prop. This makes the whole component more flexible to what you put inside it. 

If you dont know what `props.children` are then look em up on the react website it will make sense. 

TLDR: Pass the title text via `props.children` and get rid of `props.text`.

**Review by Cikica — CHANGES_REQUESTED** [review]
A bunch of small things to tighten up your method. This would be a fine component im many teams, but we cikito grande now

**Review by Cikica — APPROVED** [review]
